x-pack/plugins/observability_solution/logs_explorer/public/customizations/custom_flyout_content.tsx

```diff
@@ -55,7 +54,8 @@ const CustomFlyoutContent = ({
 
 const renderContent = ({ actions, dataView, doc }: LogsExplorerFlyoutContentProps) => (
   <EuiFlexItem>
-    <FlyoutDetail actions={actions} dataView={dataView} doc={doc} />
+    {/* TOREMOVE */}
+    {/* <FlyoutDetail actions={actions} dataView={dataView} doc={doc} /> */}
```

Comment:
Did you keep this intentionally?

Reply:
I did, it was to keep a reference on this touchpoint as I planned (and did in https://github.com/tonyghiani/kibana/pull/1/files#diff-3933fc8ad52c2ec68c912e7099f3ab7b5a291652eac29dd1271a6dc43df3223b) to remove the whole file.